[Docs-2.0] Update FAQ generate wallet (#1094)
zkSNACKs/WasabiDoc: docs/FAQ/FAQ-UseWasabi.md

## Wallet Manager

::::details
### How do I generate a new wallet?

You can [generate as many new wallets](/using-wasabi/WalletGeneration.md) as you'd like, for no extra cost and without asking for permission.

> Just click `Add Wallet` button in the bottom left corner of the main view to start the process.

Reviewer: ```suggestion
Click the `Add Wallet` button in the bottom left corner of the main view or in the searchbar.
```

![Generate new wallet in Wasabi](/AddWalletButton.png "Generate new wallet in Wasabi")

As with everything in Wasabi, you are required to label this new wallet.

> Make sure that you are precise so that you know what this wallet is for ande click `Continue`.

Reviewer: ```suggestion
Make sure that you are precise so that you know what this wallet is for and click `Continue`.
```

> Set up a password that will be used to encrypt the private key (extracted from the extended private key) on the computer.

Reviewer: ```suggestion
Add a password. It is used to encrypt the private key (extracted from the extended private key) on the computer and needed to open the wallet.
```

:::danger Careful!!
Without knowledge of the password, you CANNOT spend your bitcoin!!
:::

Make sure that you properly back up and write down this password.

> Once you've typed the same password to both rows, click `Continue`.

Reviewer: ```suggestion
Confirm the password and click `Continue`.
```

![Generate new wallet in Wasabi](/WalletNameAndPassword.png "Generate new wallet in Wasabi")

Now you will see the 12 recovery words, this is the mnemonic seed that you can use to back up and recover your wallet.
Make sure, and triple-check that you have done a proper backup of BOTH the password AND the mnemonic.
When you only have the recovery words, but not the password, then you will NOT be able to recover your wallet.
In order to protect your paper backup, consider storing the password and recovery words in two different locations.
When you've backed everything up, click `Continue`.

![Wasabi Wallet recovery words](/WalletRecoveryWords.png "Wasabi Wallet recovery words")

In this page you can do a quick check that you've written down your mnemonic seed in the correct order. 

> Once you're done, 'click `Continue` or skip the verification by clicking `Skip`.

Reviewer: skip is only on testnet

![Confirm recovery words](/ConfirmRecoveryWords.png "Confirm recovery words")

Here you get to choose your wallets coinjoin strategy.

![Wasabi Wallet coinjoin strategies](/CoinjoinStrategyView.png "Wasabi Wallet coinjoin strategies")

By clicking `Customize`, you can create a custom coinjoin strategy.

> If you do not want to set this wallet a strategy right now, dont click any of the options but just `Continue` to skip this phase for now.

Reviewer: I think you can't skip it
